tests/components/mqtt/test_discovery.py

```diff
@@ -55,7 +55,7 @@ def test_only_valid_components(mock_load_platform, hass, mqtt_mock, caplog):
     mock_load_platform.return_value = mock_coro()
     yield from async_start(hass, 'homeassistant', {})
 
-    async_fire_mqtt_message(hass, 'homeassistant/climate/bla/config', '{}')
+    async_fire_mqtt_message(hass, 'homeassistant/lock/bla/config', '{}')
```

Comment:
Please read the rest of the test and fix it accordingly.

Reply:
Yeah... sorry. Need to fix up tox to be able to run the tests locally.

Reply:
Turns out - although the docs @ https://www.home-assistant.io/docs/mqtt/discovery/ don't mention locks as being supported, they are in the source code :) Sorry for missing that. I've setup tox locally and will track things down before recommitting.

Reply:
Hi Paulus - the test still fail, but I don't have enough background knowledge on how the whole setup with Travis works to judge whether this is a temporary thing or sth. that might be related to my commits. If there is anything I can do to fix the problems or move things further along, please let me know.